mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -2039,6 +2039,60 @@ class ExtractOpFromCreateMask final : public OpRewritePattern<ExtractOp> {
   }
 };
 
+// Pattern to rewrite
+// ExtractOp(ConstantMaskOp) -> ConstantMaskOp
+// or
+// ExtractOp(ConstantMaskOp) -> ConstantOp
```

Comment:
Could you elaborate on when one rewrite is used vs the other?

Reply:
Done.